chef/cookbooks/network/recipes/default.rb

```diff
@@ -580,6 +589,14 @@ def get_datapath_id_for_ovsbridge(bridge)
       end
     end
   end
+  # If we downed a slave with wicked, bring it back under wicked's control.
+  slaves_downed_with_wicked.each do |i|
```

Comment:
I wonder if we really need ```slaves_downed_with_wicked``` here. It might just be ok to call ```wicked ifup <name>``` for all interfaces that got their configuration updated?